inject/src/main/java/io/micronaut/inject/writer/DispatchWriter.java

```diff
@@ -45,6 +46,7 @@
  * @author Denis Stepanov
  * @since 3.1
  */
+@Internal
```

Comment:
Are we ok making this class @Internal a minor release?

Reply:
It should have been internal in the first place. Everything under `io.micronaut.inject.writer` is regarded internal anyway